__tests__/api.test.js

```diff
@@ -70,9 +69,130 @@ describe('GET - /api/articles/:article_id', () => {
         );
       });
   });
-})
+
+  test('GET comment count- ', () => {
+    return request(app)
+      .get(`/api/articles/9`)
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article).toEqual(
+          expect.objectContaining({ comment_count: expect.any(String) })
+        );
+        expect(res.body.article.comment_count).toBe('2');
+      });
+  });
+});
+
+describe("6. PATCH /api/articles/:article_id", () => {
+  test("responds with status 200 and specific article is updated", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({ inc_votes: 50 })
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article).toEqual(

```

Comment:
This is not a thorough enough assertion. We should be checking what the value of the votes is if it is something that we are amending in the database. Rather than using objectContaining here I would use toEqual to specifically state what the updated object looks like.